### ISSUE TITLE
[Bug] service api declares mutually exclusive `network_mode` and `networks`: invalid compose project

### ISSUE THREAD

**Issue by ChasLui** [paulpierre/RasaGPT#1]
An error is reported after running `make install`:
~~~
🔍  Stopping any running containers .. 

service api declares mutually exclusive `network_mode` and `networks`: invalid compose project
make[1]: *** [stop] Error 15
make: *** [install] Error 2
~~~

**Comment by vorticalbox** [comment]
if you remove the `network-mode` and then update the networks at the bottom to 
```
networks:
  chat-network:
    driver: host
```

this gets things going however I am not getting

```
failed to solve: process "/bin/sh -c apt-get -y update" did not complete successfully: failed to create endpoint vqlat5g5b93a94fh37tziac1c on network bridge: failed to add the host (veth9195d8a) <=> sandbox (veth1baea51) pair interfaces: operation not supported
make[2]: *** [Makefile:115: rasa-start] Error 17```

**Comment by adamengberg** [comment]
i have the same problem.

**Comment by paulpierre** [comment]
Fixed w/ PR #2 , ty  @PythonicNinja 💪
